bind/gencpy.go

```diff
@@ -103,11 +103,24 @@ cgopy_cnv_c2py_bool(GoUint8 *addr) {
 
 static int
 cgopy_cnv_py2c_string(PyObject *o, GoString *addr) {
-	const char *str = PyString_AsString(o);
-	if (str == NULL) {
-		return 0;
+	if (PyUnicode_Check(o)) {
+		o = PyUnicode_AsUTF8String(o);
+		if (o == NULL) {
+			return 0;
+		}
+		const char *str = PyString_AsString(o);
+		if (str == NULL) {
+			return 0;
```

Comment:
add `Py_DECREF(o)` or `Py_XDECREF(o)` ? (otherwise, it seems to me we leak it.)

Reply:
There is `Py_DECREF(o)` on line 116.

Can `o` be released sooner? `str` points into `o`.